setup/releaseVersionUpdate.py

```diff
@@ -0,0 +1,46 @@
+import json
+import requests
+import sys
+import os
+from xml.dom import minidom
+from azure.storage.blob import BlobServiceClient, BlobClient, ContainerClient
```

Comment:
is this import necessary? Can you remove all extraneous imports?

Reply:
not required. Removed